src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -272,6 +272,9 @@ public override NodeInputData InputData
             Step = 1;
             Value = 1;
             ShouldDisplayPreviewCore = false;
+
+            var outport = outPorts.First();
+            outport.ToolTip = "Int64";
```

Comment:
If we do not add this, would the tooltip be "long"?

Reply:
The tooltip string is serialized as `Int32` in JSON files currently so when the DYN file is opened, it is read as `Int32`, which is not true anymore. If these files are saved again, the tooltip is correctly serialized as `Int64`.

Reply:
Thanks for the explanation